scripts/zones/Halvung/mobs/Copper_Borer.lua

```diff
@@ -4,6 +4,18 @@
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+    mob:setMobMod(xi.mobMod.AUTO_SPIKES, 1)
+
+    -- probably tops out at 50 dmg for a 75 player, max dmg seen as naked lv99 was 36
+    mob:addStatusEffect(xi.effect.BLAZE_SPIKES, 35, 0, 0)
```

Comment:
You definitely didn't test this. `xi.mobMod.AUTO_SPIKES` is used to enabled scripted spikes inside `entity.onSpikesDamage()` which you have not added to this script.

*That isn't the part that prevents them from being dispeled.*

```lua
    mob:addStatusEffect(xi.effect.BLAZE_SPIKES, 36, 0, 0)
    mob:getStatusEffect(xi.effect.BLAZE_SPIKES):setEffectFlags(xi.effectFlag.DEATH)
```
Would be correct to simply give permanent blaze spikes. That wipes the flags and sets loss to be on death only (they'd be re-applied at respawn, so that's been the pattern ever since the DSP days)

without using scripted spikes, it will fall back on the damage math the core uses for the spell blaze spikes which likely won't turn out right. If you do scripted, it'll take the base damage as an optional parameter ("damage" in the function call is this). Typically we apply an INT diff that was never documented proper (I have no idea its origin) and then resistance to its element or debuff (if any) factors in.

We have numerous examples of scripted perma spikes, this was why I directed you to one.

This is what it looks like if we copy pasta existing examples and swap it to the power you saw and switch it over to blaze spikes/fire: 
[Copper_Borer.zip](https://github.com/LandSandBoat/server/files/14170757/Copper_Borer.zip)

Reply:
if you leave it just the effect and make it non-dispelable, you'll get 36 dmg every time

if you proper script it out, it becomes heavily INT dependent unless you remove the int Diff in the script - which you can. how you handle that to ballpark it is up to you